region/norfair/east/Norfair Reserve Tank Room.json

```diff
@@ -360,6 +360,37 @@
         "Then use a second Power Bomb after waiting for the middle Dragon to finishing firing."
       ]
     },
+    {
+      "id": 32,
+      "link": [1, 4],
+      "name": "Fast Mid-air Power Bomb with Damage Boost",
+      "requires": [
+        "canTrickyJump",
+        "canMidAirMorph",
+        "canHorizontalDamageBoost",
+        {"enemyDamage": {
+          "enemy": "Dragon",
+          "type": "fireball",
+          "hits": 1
+        }},
+        {"enemyKill": {
+          "enemies": [
+            ["Dragon"],
+            ["Dragon", "Dragon"]
+          ],
+          "explicitWeapons": ["PowerBomb"]
+        }},
+        {"heatFrames": 210}
```

Comment:
this number is a wild ass guess, need help here

Reply:
Looking at the video, I measured 270 frames from door -> item, so subtracting out the 60 frames from 2 -> 4, I think this number is accurate.